### ISSUE TITLE
Array

### ISSUE THREAD

**Issue by jollen** [Squall949/ReactToDo#1]
- Use `push()` may be better than `concat()`.

**Comment by Squall949** [comment]
Why, what is the difference?

**Comment by Squall949** [comment]
I got it, thanks for the reference.

http://gunnariauvinen.com/difference-between-concat-and-push-in-javascript/

**Comment by Squall949** [comment]
For this app, I think concat is better.
Because each item is object, once user modifies one of the items,
the content of item is changed too for sure.
Also, from other resources(Stackoverflow, code school, perhaps also in doc of React), they said that return a new array is faster than changing existing array for React.

**Comment by jollen** [comment]
Yes, it is better for this case. `concat` is copying object reference to array.
